docs/getting-setup.md

```diff
@@ -136,6 +136,18 @@ and then finally test that `Secp256k1Context.isEnabled()` as in Step 2.
 
 We are finally ready to start running some programs! Follow the [instructions here](applications/server#building-the-server) to build the server. Then, follow [these instructions](applications/cli) to setup the CLI (note that this will require you install some graalvm things as detailed in the instructions).
 
+If you wish to use the GUI instead of the CLI, the setup process is the same as building the server but for the project `gui`. That is, you must run
+
+```bashrc
+sbt gui/universal:stage
+```
+
+to build and then the following (once setup is complete) to run:
+
+```bashrc
+./app/gui/target/universal/stage/bin/bitcoin-s-gui
+```
```

Comment:
I think it's easier to just have them use `sbt gui/run` that's what I always do. Then they also don't need to make sure they build a new one every update